simcon_project/templates/student_view.html

```diff
@@ -1,17 +1,16 @@
+{% load static %}
+{% load render_table from django_tables2 %}
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="description" content="Created for Simulated Conversations Senior Capstone Project">
 
-    <!-- prevent line breaks when using form fields -->
-    <style type="text/css">
-    form {display:inline; margin:0px; padding:0px;}
-    </style>
-    
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="{% static 'users/student_home.css' %}">
```

Comment:
Combining those two into one would make a mess of the static file. It's not too difficult to figure out what is affecting a particular element using your browser's inspect feature and multiple files allows for better organization I think. Otherwise everything ends up in one big pile.